lm_eval/models/huggingface.py

```diff
@@ -1031,6 +1074,11 @@ def _collate(x):
         return re_ord.get_original(res)
 
     def generate_until(self, requests: List[Instance]) -> List[str]:
+        if self.use_chat_template:
+            print(f"First element before prompt formatting...\n{requests[0].args}")
+            requests = self.tok_chat_template(requests)
```

Comment:
Think my old name for the chat templating function remained in generate_until accidentally. 

```suggestion
            requests = self.wrap_chat_template(requests)
```